lib/pg.provider.js

```diff
@@ -143,10 +143,15 @@ class PostgresProvider extends StorageProvider {
       );
 
       await Promise.all(
-        categories.map(async value => {
-          value.Id = await this.create('Category', value);
-        })
-      );
+        categories.map(async value => [
+          await this.create('Category', value),
```

Comment:
The previous version doesn't seem to have any race condition in it even though the rule says otherwise, can someone explain to me why is it that way? Or should we perhaps disable that rule?

Reply:
I think, we can disable this rule for now, since there are multiple issues with it that should be fixed before we can use it.